mitosheet/mitosheet/api/get_parameterizable_params.py

```diff
@@ -28,3 +27,27 @@ def get_parameterizable_params(params: Dict[str, Any], steps_manager: StepsManag
                 all_parameterizable_params.extend(parameterizable_params)
 
         return all_parameterizable_params
+
+def get_parameterizable_params_metadata(steps_manager: StepsManagerType) -> List[ParamData]:
```

Comment:
Almost all the code here is duplicated. Can you just get the `parametrizable_params` using the above function, and then just apply you mapping function to those? I don't think they need to be different implementations, really...

Reply:
^ this would also fix the bug you have with originally passed dataframes generating strange data from this function.